lib/VisualBuilder.js

```diff
@@ -312,6 +312,13 @@ class VisualBuilder extends EventEmitter {
     }
 
     _getLocalization(visualConfig) {
+        let apiVersionGroups = visualConfig.apiVersion.split(".");
+
+        // If API version is less than 1.6.0 it will mean that API doesn't contain schema.stringResources.json
+        if (apiVersionGroups[1] < 6) {
```

Comment:
I don't think that this is nice idea to use API version. It'd be better to check file in the directory before using this file.

Reply:
As I understood we supposed to use this approach on the yesterday meeting. Correct me if I wrong.

Reply:
please check the file in directory instead